Resources/Prototypes/Language/languages.yml

```
@@ -31,6 +31,14 @@
   - nah
   - wah
 
+# The common galactic Sign Language!
+- type: language
+  id: SignLanguage
+  signLanguage: true
+  obfuscateSyllables: false
+  replacement:
+  - "*incomprehensible*"
```

Put some things like these

```suggestion
  - signs something
  - makes weird hand gestures
```

---

Replacement is never heart its is just a placeholder for Replacement value.

---

Make it used

---

Its uses .ftl files.
I cannot translate replacement.